doc/_templates/cross_proj_navbar.html

```diff
@@ -0,0 +1,27 @@
+{# Displays links to the top-level TOCtree elements, in the header navbar. #}
```

Comment:
The comment is outdated and should be removed with the PR to avoid confusion in the future.